tekton/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - publish-images-manifests.yaml
+  - releasipeline.yaml
```

Comment:
I think these file names should be updated to 
```
build-publish-images-manifests.yaml
operator-release-pipeline.yaml
```
:thinking:

Reply:
true 👍

Reply:
done